src/python/pants/option/parser.py

```diff
@@ -573,7 +573,13 @@ def expand(val_or_str):
                 else:
                     fromfile = val_or_str[1:]
                     try:
-                        with open(fromfile) as fp:
+                        fromfile_path = Path(fromfile)
+                        fromfile_path = (
+                            fromfile_path
+                            if fromfile_path.is_absolute()
```

Comment:
I think Path(get_buildroot(), fromfile) will do what you expect if fromfile is absolute, so you don't need the is_absolute() check.

Reply:
Oh yeah, fair enough!